<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectrumViewer</class>
 <widget class="QDialog" name="SpectrumViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>350</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>View Spectrum</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources.qrc">
    <normaloff>:/icons/vespuccilogo.svg</normaloff>:/icons/vespuccilogo.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QCustomPlot" name="spectrum" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="valueLabel">
     <property name="text">
      <string>NA</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButton">
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>23</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Export...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="coordinateLabel">
     <property name="text">
      <string>(NA, NA)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="holdCheckBox">
     <property name="toolTip">
      <string>When checked, spectra are superimposed on each other. When unchecked, viewing a new spectrum resets the view.</string>
     </property>
     <property name="text">
      <string>Hold Plots</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
